doc/manual/src/glossary.md

```diff
@@ -208,6 +208,45 @@
 
     See [installables](./command-ref/new-cli/nix.md#installables) for [`nix` commands](./command-ref/new-cli/nix.md) (experimental) for details.
 
+  - [build system]{#gloss-build-system}
+
+    [build system]: #gloss-build-system
+
+    A representation of the processor architecture and platform that performs the build or compilation, typically a dash-separated string.
+
+    This is based on the [GCC system terminology](#gloss-gcc-system-terminology).
+
+    Programs that are produced by a [derivation] were configured when they were being built to have a [build system] that can be executed by the [derivation system](language/derivations.md#attr-system).
+    Conversely, when using the program after it is been built, the [build system] is no longer relevant.
```

Comment:
```suggestion
    A program that is produced by a [derivation] is built with a [build system] that can be executed by the [derivation system](language/derivations.md#attr-system).
    After it has been built, the [build system] is no longer relevant.
```

Simplifying a bit and I don't think it was a particularly important "converse" relationship.